src/schema.rs

```diff
@@ -0,0 +1,69 @@
+//! Schema
+
+#![allow(missing_docs)]
+use template;
+use std::collections::BTreeMap;
+use serde_json::Value;
+
+/// Schema
+pub struct Schema {
+    files: Vec<File>,
+}
+
+/// File
+pub struct File {
+    /// The filetype to generate
+    type_: String,
+    /// Whether to operate in fileset mode
+    fileset: bool,
```

Comment:
Is there a way to express default values for struct members? (e.g. we could default this to `false`)